plugin/__init__.py

```diff
@@ -918,6 +918,41 @@ def updateModelStyling(self, model):
         models.flush()
 
 
+    @util.api()
+    def findAndReplaceInModels(self, modelName, findText, replaceText, Front = True, Back = True, css = True):
```

Comment:
Don't do uppercase for `Front` and `Back`, no spaces around `=` in argument lists.